File: accsaber-database/src/main/java/de/ixsen/accsaber/database/repositories/model/StaffUserRepository.java
package de.ixsen.accsaber.database.repositories.model;

import de.ixsen.accsaber.database.model.staff.StaffUser;
import org.springframework.data.jpa.repository.JpaRepository;
import org.springframework.stereotype.Repository;

@Repository
public interface StaffUserRepository extends JpaRepository<StaffUser, String> {
}

File: accsaber-database/src/main/kotlin/de/ixsen/accsaber/database/repositories/model/StaffUserRepository.kt
package de.ixsen.accsaber.database.repositories.model

import de.ixsen.accsaber.database.model.staff.StaffUser
import org.springframework.data.jpa.repository.JpaRepository
import org.springframework.stereotype.Repository

@Repository
interface StaffUserRepository : JpaRepository<StaffUser, String>